### PR TITLE
UIORGS-10 Start with filters closed

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

https://issues.folio.org/browse/UIORGS-10

## Approach

replace filter config with component

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/Main/OrganizationsFilter.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+import {
+  FILTERS,
+  BOOLEAN_OPTIONS,
+  CATEGORY_OPTIONS,
+  STATUS_OPTIONS,
+  PAYMENT_METHOD_OPTIONS,
+} from './constants';
+
+class OrganizationsFilter extends Component {
```
Comment (aliaksei-chumakou) [inline src/Main/OrganizationsFilter.js]: should it be a `Component`? IMHO, SFC works fine here

Comment (NikitaSedyx) [reply src/Main/OrganizationsFilter.js]: 100%

File: src/common/constants/organization.js [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
   pending: 'Pending',
 };
+
+export const ORGANIZATION_STATUS_LABELS = {
+  [ORGANIZATION_STATUS.active]: <FormattedMessage id="ui-organizations.organizationStatus.active" />,
+  [ORGANIZATION_STATUS.inactive]: <FormattedMessage id="ui-organizations.organizationStatus.inactive" />,
+  [ORGANIZATION_STATUS.pending]: <FormattedMessage id="ui-organizations.organizationStatus.pending" />,
+};
+
+export const ORGANIZATION_PAYMENT_METHOD = {
```
Comment (aliaksei-chumakou) [inline src/common/constants/organization.js]: it looks like we have the same constants in ui-invoice. maybe it makes sense to leverage them into stripes-acq-components https://github.com/folio-org/ui-invoice/blob/d39fe77dd3511486f8c3541a0ac0b9eeddd54fba/src/common/constants/constants.js#L27

Comment (NikitaSedyx) [reply src/common/constants/organization.js]: invoices have more options

**Pull request merged**